tests/agent_server/test_agent_code_loading.py

```diff
@@ -23,10 +23,12 @@
 import tempfile
 import uuid
 from logging import DEBUG, INFO
+from typing import Sequence
```

Comment:
```suggestion
from collections.abc import Sequence
```

Reply:
I will remember this is the correct one of the two someday, but this is rather ingrained.

Reply:
I get it